### PR TITLE
config-patch

### PR DESCRIPTION
**Pull request by IchiruTake**
* Remove "makeComments" Function
* Move Learning Rate Scheduler (customized), get_optimizer(), get_callbacks as static method at Learning.py and Predict.py to config.py
* Add extra hyper-parameter for MODEL_ARCHITECTURE

### CODE REVIEW

**Review by IchiruTake — COMMENTED** [review]
PENDING
